inductiva/_cli/cmd_tasks/download.py

```diff
@@ -0,0 +1,56 @@
+"""Downloads a tasks by id via CLI."""
+import argparse
+import inductiva
+
+
+def download_task(args):
```

Comment:
Let's keep the same structure of the `tasks` module. Moreover, this is downloading the results of the simulation and not any metadata of the task.
```suggestion
def download_outputs(args):
```